Cards/CardBase.gd

```diff
@@ -32,47 +32,19 @@ func parse_card_data(card_data: Dictionary) -> void:
 	pass
 
 
-func _apply_all_effects(caster: Entity, targets: Array[Entity], effects: Array[EffectData]) -> void:
-	for target : Entity in targets:
-		for effect : EffectData in effects:
-			effect.apply_effect_data(caster, target)
+# func _apply_all_effects(caster: Entity, targets: Array[Entity], effects: Array[EffectData]) -> void:
```

Comment:
Do we still need this function since it's been commented out?

Reply:
i forgot to remove it